.github/workflows/python-app.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```

Comment:
Possibly change the name to indicate that it is testing. 

The workflow name is what appears in the badge. For example see this [raw README](https://raw.githubusercontent.com/niaid/imaris_extensions/main/README.md) and [its rendering](https://github.com/niaid/imaris_extensions/blob/main/README.md). The text in the badge is taken from [this workflow](https://github.com/niaid/imaris_extensions/blob/44c587f043bb02bae05029c97f334c7322e4eaa1/.github/workflows/main.yml#L1). Feel free to ignore as there is no badge in the README, so no-one will see the workflow name there. Up to you.